controllers/cluster_controller.go

```diff
@@ -13,7 +13,7 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-
+// +kubebuilder:rbac:groups=cluster.x-k8s.io,resources=clusters;clusters/status,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Can we move these on before the `ClusterReconciler` struct instead?

Reply:
Same comment goes for the other markers

Reply:
Done